source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,162 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect to deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :manual:`Authentication Mechanisms </core/authentication/#authentication-mechanisms>`.
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Authentication` tab.
+
+      Select your authentication method from the following options: 
+      
+      - :ref:`Username / Password <username-password>`
+      
+      - :ref:`X.509 <x509>`
+      
+      - :ref:`Kerberos <kerberos-authentication>`
+      
+      - :ref:`LDAP <ldap-authentication>`
+      
+      - :ref:`AWS IAM <aws-iam-authentication>`
+
+      .. _username-password:
+
+      Username / Password 
+      ~~~~~~~~~~~~~~~~~~~
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Authentication Mechancism:
+         
+        - Default
+
+          The ``Default`` setting selects the first authentication mechanism 
+          supported by the server according to an order of preference. For 
+          more information on the order of preference, see :driver:`Default 
+          </node/current/fundamentals/authentication/mechanisms/#default>`
```

Comment:
So this might be a case where we can try to keep users on the doc they're currently reading, rather than linking out. The order of preference seems like a core concept to the default setting, so I think we should present it here, rather than making users read it somewhere else and then come back.

Maybe we should replace the sentence that says "For more info on the order..." with something like:

```
With the ``Default`` setting, MongoDB tries to authenticate using
the following mechanisms in the order they are listed:

1. ``SCRAM-SHA-256``
2. ``SCRAM-SHA-1``
3. ``MONGODB-CR``
```
This approach has the downside of increasing maintainability, because if this order ever changes we'll also have to update this list. However I think that small burden is worth the return of keeping users on this page as best we can.

Reply:
Done.